config/passport-setup.js

```diff
@@ -26,8 +27,8 @@ passport.deserializeUser((id, next) => {
 passport.use(
   new GoogleStrategy({
     callbackURL: '/auth/google/redirect',
-    clientID: process.env.clientID,
-    clientSecret: process.env.clientSecret,
+    clientID: '831550743885-7a7f8m0t6pli22clvmresvbqg1uncdi1.apps.googleusercontent.com',
```

Comment:
sensitive information should not be tracked/public - kill these credentials, make new ones, and add them to a gitignored file